File: JAVA/1897. Redistribute Characters to Make All Strings Equal.java
class Solution {
    public boolean makeEqual(String[] w) {
        int[] f=new int[26];
        for(String s:w){
            for(int i=0;i<s.length();i++){
                f[s.charAt(i)-'a']++;
            }
        }
        for(int i=0;i<26;i++){
            if(f[i]%w.length!=0)
            return false;
        }
        return true;
    }
}

File: C++/1897. Redistribute Characters to Make All Strings Equal.cpp
class Solution {
public:
    bool makeEqual(vector<string>& w) {
        if(w.size()<2)
        return true;
        map<char,int>m;
        for(string s:w){
            for(char c:s){
                m[c]++;
            }
        }
        for(auto i=m.begin();i!=m.end();i++){
            if(i->second%w.size()!=0)
            return false;
        }
        return true;
    }
};